carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/data/CarpDataTypes.kt

```diff
@@ -53,10 +53,16 @@ object CarpDataTypes : DataTypeMetaDataMap()
 
     internal const val NON_GRAVITATIONAL_ACCELERATION_TYPE_NAME = "$CARP_NAMESPACE.nongravitationalacceleration"
     /**
-     * Acceleration along perpendicular x, y, and z axes, excluding gravity.
+     * Change in velocity along perpendicular x, y, and z axes in the device's coordinate system, excluding gravity.
      */
     val NON_GRAVITATIONAL_ACCELERATION = add( NON_GRAVITATIONAL_ACCELERATION_TYPE_NAME, "Acceleration without gravity", DataTimeType.POINT )
 
+    internal const val ACCELERATION_TYPE_NAME = "$CARP_NAMESPACE.acceleration"
+    /**
+     * Change in velocity along perpendicular x, y, and z axes in the device's coordinate system, including gravity.
```

Comment:
Again, why call it change in velocity when it's acceleration just like stated by the type name.

Reply:
Because defining a term using the term itself is circular logic. :) I was trying to add to the definition. If it is "rate of change of velocity" that's what I will change it into. Don't know where I found it stated it was simply "change in velocity". Will double-check.

Reply:
These comments should not serve as definitions of the terms imho, but rather be specific about the contents if needed in this case that it is acceleration with or without a gravitational vector.